src/app/components/navigation/TrackedRoute.tsx

```diff
@@ -19,7 +19,6 @@ import {Immutable} from "immer";
 ReactGA.initialize(GOOGLE_ANALYTICS_ACCOUNT_ID);
 ReactGA4.initialize(GOOGLE_ANALYTICS_4_MEASUREMENT_ID);
 ReactGA.set({ anonymizeIp: true });
```

Comment:
How come we have both ReactGA and ReactGA4? In the `react-ga4` repo it says that you should replace the old one:

```typescript
// Simply replace `react-ga` with `react-ga4` and remove `ReactGA.pageview()`
// import ReactGA from "react-ga";
import ReactGA from "react-ga4";
```

Reply:
Ultimately "react-ga" will be removed and GA4 will be renamed to GA, but running them concurrently at the moment until we are certain the data is feeding in correctly.  At the latest it will get changed when GA is deactivated (July), but I feel we ought to leave it here for now because it's useful to check GA vs GA4 data and see if we are getting the data we expect.